### PR TITLE
quarkus: add page

### PR DESCRIPTION
**Pull request by azthec**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented:** 3.9.2

### CODE REVIEW

File: pages/common/quarkus.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Add extension(s) to the current project:
+
+`quarkus extension add {{extension_name}}`
+
+- Build a container image using Docker:
+
+`quarkus image build docker`
+
+- Deploy application to Kubernetes:
```
Comment (kbdharun) [inline pages/common/quarkus.md:30]:
```suggestion
- Deploy the application to Kubernetes:
```

**Comment by azthec** [comment]
> Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

I've applied your suggestions!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/quarkus.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`quarkus run`
+
+- Run the current project in continuous testing mode:
+
+`quarkus test`
+
+- Add extension(s) to the current project:
+
+`quarkus extension add {{extension_name}}`
```
Comment (vitorhcl) [inline pages/common/quarkus.md:24]:
```suggestion
`quarkus extension add {{extension_1 extension_2 ...}}`
```

If I got it right, multiple extesions can be specified, right?

Also, do the extensions have some kind of ID or anything like that which makes "extension1" ambigous or not? If so, we can go with:

```suggestion
`quarkus extension add {{extension_name_1 extension_name_2 ...}}`
```

File: pages/common/quarkus.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# quarkus
+
+> Create projects, manage extensions and perform essential build and development tasks.
+> More information: <https://quarkus.io/guides/cli-tooling>.
+
+- Create a new application project:
```
Comment (vitorhcl) [inline pages/common/quarkus.md:6]:
```suggestion
- Create a new application project:
```

Does this create a new directory? Or the project is initialized on the same directory? (non-blocking change, i.e. we can merge this PR and clarify this later)

**Review by vitorhcl — APPROVED** [review]
Thank you for your contribution and welcome to tldr!

You did an excellent job, especially since it's your first contribution :)

LGTM after we my minor sugestions are applied.

File: pages/common/quarkus.md (hunk 3 above)
Comment (azthec) [reply pages/common/quarkus.md:6]: Yes, it does

**Review by vitorhcl — APPROVED** [review]
GTG after the project creation suggestion is applied

**Review by kbdharun — APPROVED** [review]

**Pull request merged**